src/main/java/com/almightyqa/TestUtils.java

```diff
@@ -0,0 +1,122 @@
+package com.almightyqa;
+
+import com.github.javafaker.Faker;
+import com.google.gson.Gson;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.stream.IntStream;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class TestUtils {
+
+    public static final Random RANDOM = new Random();
+    public static final Faker FAKER = new Faker();
+
+    public static final Gson GSON = new Gson();
+
+    public static int getRandomInt(int... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextInt(bound[0]);
+    }
+
+    public static long getRandomLong(long... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextLong(bound[0]);
+    }
```

Comment:
what about upper and lower bounds?

Reply:
just add the mi value ad use their difference as bound

Reply:
ok, so create one :)